packages/material-ui-styles/src/makeStyles/makeStyles.d.ts

```diff
@@ -41,6 +41,6 @@ export default function makeStyles<
   Props extends {} = {},
   ClassKey extends string = string
 >(
-  styles: Styles<Theme, Props, ClassKey>,
+  styles: Styles<Theme, Props, ClassKey> | Record<ClassKey, string>,
```

Comment:
Should update `@material-ui/core` as well
https://github.com/mui-org/material-ui/blob/5dc2402c9df6f148756129bdabdb1e2fd8363713/packages/material-ui/src/styles/makeStyles.d.ts#L11